src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -38,6 +38,8 @@ public static partial class PlatformDetection
         public static bool IsNetBSD => RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"));
         public static bool IsAndroid => RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"));
         public static bool IsNotAndroid => !IsAndroid;
+        public static bool IsAndroidX64 => IsAndroid && Is64BitProcess;
```

Comment:
Should this be `IsX64Process`? If its just `Android X64`? `Is64BitProcess` would apply to both X64 and Arm64 right

Reply:
Plus it would be more akin to the `IsAndroidX86` right below

Reply:
I think you're right. Good catch!